assembly/onpremises-ide-compiling-war-ide/pom.xml

```diff
@@ -27,8 +27,8 @@
     <name>Onpremises IDE :: WAR :: Compiling App</name>
     <properties>
         <generated.sources.directory>${project.build.directory}/generated-sources/gen</generated.sources.directory>
-        <maven.compiler.source>1.7</maven.compiler.source>
-        <maven.compiler.target>1.7</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
I removed these properties. They will be used from [che-parent](https://github.com/eclipse/che-parent/blob/master/pom.xml#L121)